src/main/java/com/example/java/test/junior/developer/model/Category.java

```diff
@@ -27,5 +28,10 @@ public class Category {
     @JoinColumn(name = "language_id")
     private Language language;
 
+    @OneToMany(cascade = CascadeType.DETACH,
+            mappedBy = "category",
+            fetch = FetchType.EAGER)
```

Comment:
Are you sure EACH time you fetch category - you want to fetch Questions from DB? 
According to design that I remember - you need languages+categories for main screen. But questions - you need them only when user selected category and started quiz. Why to fetch that all the time?